src/main/java/com/revature/services/SellerServiceImpl.java

```diff
@@ -0,0 +1,47 @@
+package com.revature.services;
+
+import com.revature.models.Seller;
+import com.revature.repositories.SellerRepo;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class SellerServiceImpl implements SellerService {
+
+    @Autowired
+    SellerRepo sr;
+
+    @Override
+    public Seller addSeller(Seller seller) {
+        return sr.save(seller);
+    }
+
+    @Override
+    public Seller getSellerById(int id) {
```

Comment:
We probably want to return an Optional here instead of returning an empty object. Makes checking for it easier in the controller.